include/sensor.h

```diff
@@ -31,12 +31,18 @@ extern "C" {
  * @brief Representation of a sensor readout value.
  *
  * The value is represented as having an integer and a fractional part,
- * and can be obtained using the formula val1 + val2 * 10^(-6).
+ * and can be obtained using the formula val1 + val2 * 10^(-6). Negative
+ * values also adhere to the above formula, but may need special attention.
+ * Here are some examples of the value representation:
+ *     0.5: val1 =  0, val2 =  500000
```

Comment:
to get this formatted the way you probably intended, add a blank line before this indented section, and use at least 4 spaces (relative to the lines before) for the indented lines (the first example line has 4 spaces but the others only have 3, so give them all an extra space):

     * Here are some examples of the value representation:
     *
     *      0.5: val1 =  0, val2 =  500000
     *     -0.5: val1 =  0, val2 = -500000
     *     -1.0: val1 = -1, val2 =  0
     *     -1.5: val1 = -1, val2 = -500000

Reply:
Thanks, updated, directly copied from the snippet above.